en.txt

```diff
@@ -0,0 +1,67 @@
+About me:
+	I mostly have experience with Linux servers, Unix, c programming language
+	I like Implementing interesting ideas.
+	I have worked briefly in 2 startups, here is what I did:
+	Start-up 1 - inn0web, website: https://inn0web.com, (time period 2022.08 - 2022.10):
+		- I wrote a bot for finding bicycles of a specific price, in particular locations.
+		When found, send information to Telegram.
+		- The bot is designed to find the most popular Reddit posts, and according to the topic, make 10 min
+		video montage to upload them to different YouTube channels
+	Start-up 2 - Monopulse, website: https://www.monopulse.eu, (time period ~2023.02): 
+		- I was assigned a task to work with embedded Linux in the yacto project.	
+	~2023.09 Created service to help transfer all Etsy products to WordPress using Etsy & Woocommerce API
+
+	NOTE: The source code of these projects is not public.
+
+Java:
+	I worked with a professional Java mentor who assigned me Java tasks, 
+	which were reviewed, corrected, and completed through a process involving 
+	Jira for task management, Git for version control, and code reviews
+	conducted via Pull Requests. 
+	You can see the entire tasking process in the GitHub links below.
+		https://github.com/Adomo-Software/java-fix-me
+		https://github.com/Adomo-Software/rating-system
+		https://github.com/Adomo-Software/caesars-palace-casino
+		https://github.com/Adomo-Software/cherry-tree-hotel
+		https://github.com/Adomo-Software/continental-hotel
+		https://github.com/Adomo-Software/hotel
+	Tasks lists:
+		https://algirdasalisauskas.atlassian.net/jira/software/projects/JBH/list
+		https://algirdasalisauskas.atlassian.net/jira/software/projects/JC/list
+		https://algirdasalisauskas.atlassian.net/jira/software/projects/JFM/list
```

Comment:
Better naming is needed and point to the boaed instead of list. For example:
[Java basics: Hotel](https://algirdasalisauskas.atlassian.net/browse/JBH)
[Java Collections](https://algirdasalisauskas.atlassian.net/browse/JC)
[Java Fix Me](https://algirdasalisauskas.atlassian.net/browse/JFM)